### DISCUSSION TITLE
Where is "Relate with HelpDesk" action in Mail Scanner on YT 7.0.1

### DISCUSSION THREAD

**Discussion by zielonekrzeslo** [YetiForceCompany/YetiForceCRM#17475] [How to]
Hi,
in 6.5 version was "Relate with HelpDesk" action in list and on 7.0.1 was disapper. 
In the "modules\OSSMailScanner\scanneractions" directory there is no BindHelpDesk.php file, the directory is empty.
I need to link an email with a ticket for a account.
Where I can find this option in new version?

![obraz](https://github.com/YetiForceCompany/YetiForceCRM/assets/166414275/003ae86c-6d52-4613-b258-cf227526d7ec)
![obraz](https://github.com/YetiForceCompany/YetiForceCRM/assets/166414275/5e5bcdbb-9d1e-4358-952c-0d6268fe508d)

**Comment by rskrzypczak** [comment]
Just set the appropriate value in the `Search fields` field. The system will look for the association.
In version 7 there are no limitations to single modules, you can link an email to any module that has relationships with Email history.
![image](https://github.com/user-attachments/assets/5c6ee912-4416-40a2-b2c0-dc4508fd72cf)
